build/simp-adapter.spec

```diff
@@ -12,6 +12,7 @@ Buildarch: noarch
 
 Prefix: %{_sysconfdir}/simp
 
+Requires: git
 Requires: rsync
 Requires(post): puppet
 Requires(post): puppetserver
```

Comment:
@jeefberkey What about just not dropping the PE ones into the public download buckets? The main reason that we do it is to ensure that we have a compatible version of puppet installed.

Alternatively, I suppose that we could just require `puppet` of the correct version and call it done.

Reply:
That said, I think that the suggestion should be a different PR whichever way we go.

Reply:
both puppetserver and puppet/puppet-agent shouldn't be requirements of simp-adapter. this is supposed to be the rpm installation wrapper to copy files into the correct place. simp.spec can declare it's dependency on puppetserver or puppet-agent.

Reply:
This sounds like the trail for a decision log since this is specifically one of the first things that `simp-adapter` was created to do. This may be in error but it functionally has nothing to do with this PR as it stands and should be a new ticket and additional work.

Reply:
Relevant [Decision Log](https://simp-project.atlassian.net/wiki/spaces/SD/pages/719814664/Where+should+the+requirement+for+Puppet+and+or+PE+be+housed+in+the+SIMP+RPM+stack) started for follow up